samples/tab-external-auth/csharp/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-tab-external-auth-csharp",
```

Comment:
This name looks wrong, please verify readme.

Reply:
The name is similar in format to other-samples
![image](https://github.com/OfficeDev/Microsoft-Teams-Samples/assets/87962396/f3630979-554d-4f1f-b37c-871eed2b0c78)